identifiers/country-in/districts.csv

```diff
@@ -692,6 +692,26 @@ ocd-division/country:in/territory:dl/district:se,South East Delhi
 ocd-division/country:in/territory:dl/district:sh,Shahdara
 ocd-division/country:in/territory:dl/district:sw,South West Delhi
 ocd-division/country:in/territory:dl/district:wd,West Delhi
+ocd-division/country:in/territory:jk/district:an,Anantnag
+ocd-division/country:in/territory:jk/district:bn,Bandipora
+ocd-division/country:in/territory:jk/district:br,Baramulla
+ocd-division/country:in/territory:jk/district:bu,Budgam
+ocd-division/country:in/territory:jk/district:do,Doda
+ocd-division/country:in/territory:jk/district:ga,Ganderbal
+ocd-division/country:in/territory:jk/district:ja,Jammu
+ocd-division/country:in/territory:jk/district:ka,Kathua
+ocd-division/country:in/territory:jk/district:ki,Kishtawar

```

Comment:
Should this be spelled `Kishtwar`? https://en.wikipedia.org/wiki/Kishtwar_district

Reply:
You are right! Corrected.